src/cloud/project/privatelink-service.md

```diff
@@ -0,0 +1,138 @@
+   -  **Cloud region**–Provide the Cloud region where the account is hosted for verification purposes
+   -  **Services and communication ports**–Magento must open ports to enable service communication between VPCs, for example _Webserver, HTTP port 80_, _SFTP port 2222_
+   -  **Magento Cloud Project ID**
+   -  **Connection Type**–Specify unidirectional or bidirectional for connection type
+   -  **Service Name**–Service name for the VPC service endpoint that Magento must connect to. Required only to enable bidirectional PrivateLink connections.
+
+### Enablement workflow
+
+The following workflow outlines the PrivateLink enablement process for {{site.data.var.ece}}.
+
+1. **Customer** contacts their Magento CSM to request PrivateLink enablement, providing the customer data required for PrivateLink enablement.
+
+1. After verifying the customer data, **the Magento CSM** submits a Magento Support ticket request for PrivateLink enablement.
+
+   We use the Support ticket to coordinate communication between you, your CSM, the Magento infrastructure team, and the Magento Support team during the enablement process.
+
+1. **Magento infrastructure team** enables customer account access to the Magento VPC.
+
+   -  Update the Magento VPC configuration to accept requests initiated from the customer Cloud platform account.
+
+   -  Update the Support ticket to provide Magento VPC connection details to the customer.
+
+1. **Customer** submits a PrivateLink connection request to the Magento VPC using the Magento VPC connection details.
+
+1. **Magento infrastructure team** approves the connection request.
+
+1. After connection request approval, **the customer** verifies the connection between your VPC and the Magento VPC.
+
+1. Additional steps to enable bidirectional connections:
+
+   -  **Magento infrastructure team** uses the endpoint service name supplied by the customer to submit a connection request from the Magento VPC to the customer Network Load Balancer (NLB).
+
+   -  **Customer** approves the request to complete the setup. See the Cloud platform documentation for instructions:
+
+      -  For AWS, see [Accepting and rejecting interface endpoint connection requests][].
+      -  For Azure, see [Manage connection requests][].
+
+   -  **Customer** verifies the connection to and from the Magento VPC.
+
+## Customer set up for bidirectional PrivateLink communication
+
+The customer VPC must have a the following resources available to support bidirectional PrivateLink connections:
+
+-  A Network Load Balancer
+-  An endpoint service configuration that enables access to a service from your VPC environment
+-  An [interface endpoint][] (AWS) or [private endpoint][] (Azure) that allows Magento to connect to endpoint services hosted in your VPC
+
+If these resources are not available in the customer VPC, you must sign into your Cloud platform account to add the configuration.
+
+-  Amazon VPC console– ```https://console.aws.amazon.com/vpc/```
```

Comment:
Not sure why code block ticks are used here. Best to use single ticks unless they are posing a strange problem.